src/app/slotting/edit/edit.service.ts

```diff
@@ -48,6 +48,43 @@ export class EditService {
     return ['zeus', 'air', 'armor', 'art', 'hq', 'inf', 'maint', 'mech_inf', 'med', 'mortar', 'motor_inf', 'plane', 'recon', 'service', 'support', 'uav'];
   }
 
+  public get slotPresets(): Array<{ shortcode: string, description: string }> {
+    return [
+      { shortcode: 'CL', description: 'Company Leader' },
+      { shortcode: 'PTL', description: 'Platoon Leader' },
+      { shortcode: 'PTA', description: 'Asst. Platoon Leader' },
+      { shortcode: 'PTM', description: 'Platoon Medic' },
+      { shortcode: 'SQL', description: 'Squad Leader' },
+      { shortcode: 'SQA', description: 'Asst. Squad Leader' },
+      { shortcode: 'SQM', description: 'Squad Medic' },
+      { shortcode: 'FTL', description: 'Fireteam Leader' },
+      { shortcode: 'LAT', description: 'Light Anti-Tank Gunner' },
+      { shortcode: 'AT', description: 'Anti-Tank Gunner' },
```

Comment:
ich meine daß hier eher der Begriff [Anti-Tank Rifleman](https://shacktac.fandom.com/wiki/Anti-Tank_Rifleman) benutzt wird

Reply:
Ja aber heißt ja auch Machine Gunner und Anti-Tank Gunner 🤔 Also das was ich mir sonst überlegt hab war _"Rifleman (AT)"_ (so heißt es in Arma) passt halt ned zur Abkürzung.

Reply:
es heißt "machine gunner" weil der typ ne machine gun bedient, die heißt nunmal machine gun, während der AT gunner nur dann ein AT gunner wäre wenn er eine Anti tank gun bediente, was er nicht tut.

Reply:
Problem is, dass `AT Rifleman` bzw. `Rifleman (AT)` (oder jede andere Form davon) wird oft mit dem LAT also Rifleman mit Einmal Werfer gleichgesetzt.
Wenn ich jetzt den einen also `Rifleman (AT)` und den anderen `Anti-Tank Rifleman` nenn, dann is doof. 

Vorschlag: 
- `Anti-Tank Specialist` für AT
- `Rifleman (AT)` für LAT

Reply:
Ist umbenannt